ui/src/components/HookForm/index.jsx

```diff
@@ -748,9 +748,7 @@ export default class HookForm extends Component {
                             window.env.TASKCLUSTER_ROOT_URL,
                             'reference/core/hooks/firing-hooks'
                           )}>
-                          {'"'}
-                          firing hooks
-                          {'"'}
+                          &quot;firing hooks&quot;
```

Comment:
This was changed recently in https://github.com/taskcluster/taskcluster/commit/3da927a303f5e7062780f71bea6648f74626ccb9. The quotations were removed so this won't be needed anymore.